src/test/java/teammates/common/datatransfer/questions/FeedbackTextResponseDetailsTest.java

```diff
@@ -0,0 +1,53 @@
+package teammates.common.datatransfer.questions;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.SanitizationHelper;
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackTextResponseDetails}.
+ */
+public class FeedbackTextResponseDetailsTest extends BaseTestCase {
```

Comment:
I think we can test `setAnswer` and also the default constructor (that sets answer to empty string)